tensorrt_llm/_torch/autotuner.py

```
@@ -1059,12 +1059,22 @@ def _profile_runners(
-                    tactic=-1,
-                    do_preparation=True,
-                    **kwargs,
-                )
+                try:
+                    runner(
+                        input_tensors,
+                        tactic=-1,
+                        do_preparation=True,
+                        **kwargs,
+                    )
+                except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)